### ISSUE TITLE
Top navigational banner with links

### ISSUE THREAD

**Issue by key2market** [key2market/dataviz#20]
At the top of the Wiz, where we have "Save Chart Restore Chart" lets create a menu that will have a link to Dashboards as well. So there will be 3 links now:
Save Chart
Restore Chart
Dashboards -> This will have a dropdown with links to the dashboards that the user has saved.
Help (right now this will lead to nowhere)

Could we please make the spacing between the links smaller, so it looks like a normal menu? I am attaching example from chrome.
![screen shot 2016-01-21 at 2 15 12 pm](https://cloud.githubusercontent.com/assets/16460237/12481439/787d2bac-c049-11e5-938b-17dc8025d9df.png)

**Comment by prahladyeri** [comment]
I've added `Milestone 1.0` tag to this and other enhancements just created. The thing with milestones is that it will help us classify and prioritize some feature requests over other. Github will also show you stats regarding the [completion status](https://github.com/key2market/dataviz/milestones) of Milestones. Please let me know if this is all right.

**Comment by prahladyeri** [comment]
The menu has been built accordingly (I've tried to replicate Chrome menu as much as possible, but feel free to suggest color changes). However, the following cannot be implemented out of the box:

> This will have a dropdown with links to the dashboards that the user has saved.

For that the functionality of "saving a dashboard" needs to be developed, which doesn't exist as of now. What exists right now is just the `dash.html` where a user can just *view* the saved charts by clicking the plus button. But no worries, I've created this separate issue #23 for the same. We will track it from there.
